pom.xml

```diff
@@ -74,8 +74,8 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.289.x</artifactId>
-                <version>1090.v0a_33df40457a_</version>
+                <artifactId>bom-weekly</artifactId>
```

Comment:
can this be run on the 2.319 LTS line instead of the weekly?

Reply:
Downgraded the bom 👍🏻